charm_versions.md

```diff
@@ -0,0 +1,138 @@
+# Versions
+
+## Juju Version
+
+The Juju version uses [semantic versioning](https://semver.org/), so is a string in the format
+major.minor.patch, where each section is a positive integer (which may be higher than 9).
+
```

Comment:
Feels like something worth adding to https://juju.is/docs/juju/juju-version-compatibility-matrix , which should maybe be rebranded as well. PS What do we mean by Juju here: the client, the agent, both? PS The published doc isn't clear on that either.